tests/e2e/unified_test.go

```diff
@@ -208,7 +208,12 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 					// We want to use SSA everywhere, but some of our tests are broken by SSA
 					switch group := primaryResource.GetObjectKind().GroupVersionKind().Group; group {
 					case "bigtable.cnrm.cloud.google.com",
-						"orgpolicy.cnrm.cloud.google.com":
+						"orgpolicy.cnrm.cloud.google.com",
+						"backupdr.cnrm.cloud.google.com",
+						"datastream.cnrm.cloud.google.com",
+						"gkebackup.cnrm.cloud.google.com",
+						"speech.cnrm.cloud.google.com",
+						"vmwareengine.cnrm.cloud.google.com":
```

Comment:
This would increase the maintenance overhead more and more. Can we at least keep the list in its own file? Another question, how did you get the list?

Reply:
These are just the API groups that I recently worked on, so I am sort of familiar with the resources and test cases.

Reply:
I think we can keep adding the API groups and switch to use "exclude list" when we have most API groups enabled on SSA.